src/modules/fancyzones/editor/FancyZonesEditor/EditorWindow.cs

```diff
@@ -13,7 +13,7 @@ public class EditorWindow : MetroWindow
     {
         protected void OnSaveApplyTemplate(object sender, RoutedEventArgs e)
         {
-            EditorOverlay mainEditor = EditorOverlay.Current;
+            EditorOverlay mainEditor =  App.Overlay[MonitorVM.CurrentMonitor];
```

Comment:
nit: Copypasta in this file is coming with an extra whitespace in the beginning.
Here, and in the other 2 occurrences.